ehsm_kms_service/ehsm_kms_params.js

```diff
@@ -72,6 +72,48 @@ const ehsm_kms_params = {
     },
     aad,
   },
+  GenerateDataKeyWithoutPlaintext: {
+    cmk_base64: {
+      type: 'string',
+      maxLength: 1024,
```

Comment:
the cmk's max length is not 1024, it's decided by the keyspec, for now:
if the keyspec=AES128, then the length is 64,  if the keyspec=RSA3072, then the length seems is 2168

Reply:
same issue for the above APIs define

Reply:
it will be fix

Reply:
you can just set the hardcode max length for now(e.g. 4K, don't set the same length with plaintex data), maybe later we can refine the cmk's parameter to the cmk's ID which is a fix length.